pd/src/staking.rs

```diff
@@ -1,12 +1,32 @@
+use num::bigint;
+use num::rational;
+use penumbra_crypto::{Address, Value};
+use serde::{Deserialize, Serialize};
 use tendermint::{vote, PublicKey};
 
 const PENUMBRA_BECH32_VALIDATOR_PREFIX: &str = "penumbravalpub";
 /// Validator tracks the Penumbra validator's long-term consensus key (tm_pubkey), as well as their
 /// voting power.
-#[derive(Debug, Eq, PartialOrd, Ord, Clone, Copy)]
+#[derive(Deserialize, Serialize, Debug, Eq, Clone)]
 pub struct Validator {
+    /// tm_pubkey is the validator's long-term Tendermint public key, where the private component
+    /// is used to sign blocks.
     tm_pubkey: PublicKey,
+
+    /// voting_power is the validator's current voting power.
```

Comment:
Style nit: since Rustdoc will always display the doc string in the context of the item it attaches to, here and on the other fields it'd be better to structure the doc without the name, like
```suggestion

    /// The validator's current voting power.
```

Reply:
oh neat!